deployment/scripts/upgrade.sh

```diff
@@ -0,0 +1,129 @@
+#! /bin/bash
+
+set -ex
+
+fluentd_nodeselector=${FLUENTD_NODESELECTOR:-logging-infra-fluentd=true}
+TIMES=300
+
+fluentd_nodes=""
+fluentd_nodeselector=""
+
+function getSvcVersion {
+  set +e
+  local version=`oc get svc logging-es -o jsonpath='{.metadata.annotations.LOGGING_VERSION}' 2>1`
```

Comment:
oops, thanks for the catch!

and that makes more sense to do; i'll make that change too